test/services/licences/view-licence-contact-details.service.test.js

```diff
@@ -51,6 +63,12 @@ describe('View Licence Contact Details service', () => {
 
       expect(result).to.equal({
         activeTab: 'contact-details',
+        customerContacts: [{
+          communicationType: 'Additional Contact',
+          email: 'dfd@email.com',
+          name: 'Donald Duck'
+        }
+        ],
```

Comment:
NIT - Just being consistent with the start of the declaration

```suggestion
        }],
```